openmetadata-spec/src/main/resources/json/schema/entity/services/connections/dashboard/supersetConnection.json

```diff
@@ -11,6 +11,30 @@
       "type": "string",
       "enum": ["Superset"],
       "default": "Superset"
+    },
+    "apiConnection":{
+      "title": "API Connection",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "title": "Provider",
+          "description": "Authentication provider for the Superset service. For basic user/password authentication, the default value `db` can be used. This parameter is used internally to connect to Superset's REST API.",
+          "type": "string",
+          "default": "db"
```

Comment:
if we have the list of providers, we can add an enum here..just so we are strict about what the users can pass